<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMenuBarMapEditor</class>
 <widget class="QMenuBar" name="WidgetMenuBarMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="nativeMenuBar">
   <bool>false</bool>
  </property>
  <widget class="QMenu" name="menuFloors">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="tearOffEnabled">
    <bool>false</bool>
   </property>
   <property name="title">
    <string>Floor</string>
   </property>
   <property name="toolTipsVisible">
    <bool>false</bool>
   </property>
   <addaction name="actionFloors"/>
  </widget>
  <widget class="QMenu" name="menuEvents">
   <property name="title">
    <string>Object</string>
   </property>
  </widget>
  <widget class="QMenu" name="menuFace_Sprite">
   <property name="title">
    <string>Face Sprite</string>
   </property>
   <addaction name="actionFace_Sprite"/>
   <addaction name="actionFix_Sprite"/>
   <addaction name="actionDouble_Sprite"/>
   <addaction name="actionQuadra_Sprite"/>
   <addaction name="actionWall_Sprite"/>
  </widget>
  <action name="actionFloors">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/floor.png</normaloff>:/icons/Ressources/floor.png</iconset>
   </property>
   <property name="text">
    <string>Floor</string>
   </property>
  </action>
  <action name="actionEvents">
   <property name="text">
    <string>Objects</string>
   </property>
  </action>
  <action name="actionFace_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_face.png</normaloff>:/icons/Ressources/sprite_face.png</iconset>
   </property>
   <property name="text">
    <string>Face Sprite</string>
   </property>
  </action>
  <action name="actionFix_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_fix.png</normaloff>:/icons/Ressources/sprite_fix.png</iconset>
   </property>
   <property name="text">
    <string>Fix Sprite</string>
   </property>
  </action>
  <action name="actionPencil">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/pencil.png</normaloff>:/icons/Ressources/pencil.png</iconset>
   </property>
   <property name="text">
    <string>Pencil</string>
   </property>
  </action>
  <action name="actionTin_of_paint">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/pin.png</normaloff>:/icons/Ressources/pin.png</iconset>
   </property>
   <property name="text">
    <string>Tin of paint</string>
   </property>
  </action>
  <action name="actionDouble_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_double.png</normaloff>:/icons/Ressources/sprite_double.png</iconset>
   </property>
   <property name="text">
    <string>Double Sprite</string>
   </property>
  </action>
  <action name="actionQuadra_Sprite">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_quadra.png</normaloff>:/icons/Ressources/sprite_quadra.png</iconset>
   </property>
   <property name="text">
    <string>Quadra Sprite</string>
   </property>
  </action>
  <action name="actionWall_Sprite">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/icons/Ressources/sprite_walls.png</normaloff>:/icons/Ressources/sprite_walls.png</iconset>
   </property>
   <property name="text">
    <string>Wall Sprite</string>
   </property>
  </action>
  <action name="actiondfsfffffffffffffffffffffffff">
   <property name="text">
    <string>dfsfffffffffffffffffffffffff</string>
   </property>
  </action>
  <addaction name="menuFloors"/>
  <addaction name="menuFace_Sprite"/>
  <addaction name="menuEvents"/>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
